server/src/test/java/io/crate/integrationtests/NotPredicateIT.java

```diff
@@ -0,0 +1,32 @@
+package io.crate.integrationtests;
+
+import static io.crate.testing.Asserts.assertThat;
+
+import org.elasticsearch.test.IntegTestCase;
+
+public class NotPredicateIT  extends IntegTestCase {
```

Comment:
Do you plan to keep this? I guess we don't need this, if we have the tests in `ThreeValuedLogicQueryBuilderTest`

Reply:
👍